pkg/virt-handler/vm.go

```diff
@@ -817,6 +827,14 @@ func (d *VirtualMachineController) defaultExecute(key string,
 		log.Log.Info("Domain does not exist")
 	}
 
+	// make sure the per-vmi shared directory exists
+	interfaceCacheFile := "/var/run/kubevirt-network/%s/interface-cache-%s.json" // todo
+	dir := fmt.Sprintf(path.Dir(interfaceCacheFile), string(vmi.UID))
+	err := os.MkdirAll(dir, 0755)
```

Comment:
this seems like logic that belongs in the "Sync" function ```processVMUpdate``` instead of here.

Reply:
This is not needed at all anymore, thanks for spotting.